Tomas/script.js

```diff
@@ -0,0 +1,38 @@
+let log = console.log;
+
+// Write a function that accepts two numbers and validate that they are numbers.
+// After that, the function should print y exponential values starting from x
+const exponentialsOfX = [];
+function checkIfNumber(x, y) {
+  if (typeof x == "number" && typeof y == "number") {
+    log(x + " and " + y + " are numbers");
+    for (i = 1; i <= y; i++) {
+      exponentialsOfX.push(Math.pow(x, i));
+    }
+    log(`exponential growth of ${x} goes like: ${exponentialsOfX.join(", ")}`);
+  } else {
+    log(
+      "All the time in the world we have. Submit two numbers you once, continue we will."
+    );
+  }
+}
+checkIfNumber(5, 7);
+
+// Create a function named printFavoriteFruit. Declare a variable outside of the function and store your favorite fruit as a value. Reassign the variable within the function and print "My favorite fruit is: x".
+let favoriteFruit = "kiwi";
+function printFavoriteFruit() {
+  log(`My favorite fruit is ${favoriteFruit}.`);
```

Comment:
Task 2: Reassign the variable that you made within the function and then try to print the result